cypress/obj/variables.js

```diff
@@ -0,0 +1,59 @@
+class Variables {
+
+    rootPage(){
+        cy.visit('/')
+    }
+    getUserValidationIndicator(){
+        return cy.get('.Select.not-valid')
+    }
+    submit(){
+        return cy.get('[type="submit"]')
+    }
+    loginFormIdList(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+    loginRoleIdList(){
+        return cy.get('[id="loginForm.role"]')
+    }
+    loginUserName(userName){
+        return cy.get('[aria-label="'+userName+'"]')
+    }
+    loginRoleName(role){
+        return cy.get('[aria-label="'+role+'"]')
+    }
+    getUrl(){
```

Comment:
And at this point it should be in CalendarPage, or TimeLoggingPage or any other name page.